openwisp_utils/qa.py

```diff
@@ -228,7 +228,7 @@ def _find_issue_mentions(message):
      - count of mentions performed correctly
        (using one of the common github keywords)
     """
-    words = message.split(' ')
+    words = message.split()
```

Comment:
By removing quotes from here, words will be splitted for any white space (carriage return, new line etc too)

Reply:
I hadn't noticed this before, thanks!